pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TopicsConsumerImplTest.java

```diff
@@ -539,6 +540,38 @@ public void testTopicNameValid() throws Exception{
         }).get();
     }
 
+    @Test
+    public void testExclusiveSubscribe() throws Exception {
+        final String topicName = "persistent://prop/use/ns-abc/testTopicNameValid";
```

Comment:
It's better not to use the V1 version topic name. We will remove it in the future.

Reply:
I think it's consistent with the other tests in `TopicsConsumerImplTest`.

Reply:
I will fix